mbedtls-sys/Cargo.toml

```diff
@@ -27,9 +27,12 @@ libc = { version = "0.2.0", optional = true }
 libc = { version = "0.2.0" }
 
 [build-dependencies]
-bindgen = "0.19.0"
+bindgen = { git = "https://github.com/rust-lang/rust-bindgen/", branch = "master" }
```

Comment:
I guess this will be updated to the new bindgen release before merging?

Reply:
Yes, as mentioned above.